spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -124,6 +128,26 @@
         get :index, deposit_stats: {start_date: 1.days.ago.strftime("%Y-%m-%d"), end_date: 0.days.ago.strftime("%Y-%m-%d")}
         expect(assigns[:depositors]).to include({ key: user1.user_key, deposits: 1, user: user1}, { key: user2.user_key, deposits: 1, user: user2 })
       end
+
+      context "more than 10 users" do
+        let(:users) { [] }
+        before do
+          (1..12).each do |number|
+            luser = User.create(email: "user#{number}@blah.com", password: "blahbalh")
+            users << luser
+            GenericFile.new(id: "more#{number}") do |gf|
+              gf.apply_depositor_metadata(luser)
+              gf.update_index
+            end
+          end
+        end
+
+        it "gathers user deposits" do
+          get :index
+          expect(assigns[:depositors]).to include({ key: user1.user_key, deposits: 2, user: user1 }, { key: user2.user_key, deposits: 1, user: user2 })
```

Comment:
@cam156 Ah, I see. And `user1` is not one of the 12 created users. Those are tested on the line below. Gotcha!